explainer.md

```diff
@@ -287,23 +300,24 @@ interaction with the PEPC (for example, user agents generally allow users to
 control permissions on various UI surfaces that are entirely separate from the
 site's rendering area). Therefore events specific to the PEPC will only deal
 with the user's actions on the Permission UI, and specifically with the user
-closing it either by dismissing it or by taking some other action on it that 
+closing it either by dismissing it or by taking some other action on it that
 causes it to close (e.g. they accept it). This allows sites to respond to this
 event by providing more context to potentially help the user make a decision.
 These two events will be added to
 [GlobalEventHandlers](https://html.spec.whatwg.org/#globaleventhandlers) and can
 only target `permission` HTML elements. They do not bubble and are not
 cancelable.
 
-- `ondismiss` - raised when the permission UI triggered by the PEPC has
-  been dismissed by the user (for example via clicking the 'x' button or
-  clicking outside the prompt)
-- `onresolve` - raised when the permission UI triggered by the PEPC has
-  been resolved by the user taking some action on the prompt itself. Note that
-  this does not necessarily mean the permission state has changed, the user
-  might have taken an action that maintains the status quo (such as an action
-  that continues allowing a permission on a
-  [previously granted](#ui-when-the-permission-is-already-granted) type of UI).
+-   `ondismiss` - raised when the permission UI triggered by the PEPC has been
+    dismissed by the user (for example via clicking the 'x' button or clicking
+    outside the prompt)
+-   `onresolve` - raised when the permission UI triggered by the PEPC has been
```

Comment:
```suggestion
-   `onresolve` - fired when the permission UI triggered by the PEPC has been
```

Reply:
"resolve" is very much "promise terminology"... might be good to find an alternative even name.

Reply:
To be fixed in follow-up PR